am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -219,6 +232,38 @@ public FilteredResourceEnvelope filterResource(@NotBlank String userId,
             .build();
     }
 
+    private Map<JsonPointer, Set<Permission>> filterAttributePermission(
+        Map<JsonPointer, Set<Permission>> attributePermissions, Set<JsonPointer> visibleAttributes) {
+
+        return attributePermissions.entrySet().stream()
+            .filter(entry -> visibleAttributes.contains(entry.getKey()))
+            .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+    }
+
+    @SuppressWarnings("LineLength")
+    private Set<JsonPointer> getFilteredAttributesBySecurityClassification(@NotEmpty Set<@NotBlank String> userRoles,
+                                                                            @NotEmpty @Valid Map<JsonPointer, SecurityClassification>
+                                                                                attributeSecurityClassification) {
+        final Set<SecurityClassification> securityClassifications =
+            SecurityClassifications.getVisibleSecurityClassifications(getMaxSecurityRole(userRoles));
+
+        return attributeSecurityClassification.entrySet()
+            .stream()
```

Comment:
put .stream() on line above

Reply:
Moved the .stream() to the appropriate line.